src/Mod/TechDraw/Gui/QGIView.cpp

```diff
@@ -580,7 +565,7 @@ QGIView* QGIView::getQGIVByName(std::string name)
     QList<QGraphicsItem*>::iterator it = qgItems.begin();
     for (; it != qgItems.end(); it++) {
         QGIView* qv = dynamic_cast<QGIView*>((*it));
-        if (qv) {
+        if (!qv) {
```

Comment:
@WandererFan bug introduced. Was meant to fix code, not the opposite. I wanted to do a fix PR but literally got 1% battery left. Thanks